hdr_schemata/models/HDRUK/v3_0_0/FormatAndStandards.py

```diff
@@ -11,20 +11,24 @@ class FormatAndStandards(BaseModel):
     class Config:
         extra = "forbid"
 
-    vocabularyEncodingScheme: List[ControlledVocabulary] = Field(
-        ..., 
-        **an.vocabularyEncodingScheme.__dict__, 
+    vocabularyEncodingScheme: List[ControlledVocabularyEnum] = Field(
```

Comment:
Could you run `create_markdown.py` to regenerate the `3.0.0.form.json` too?  That should fix these options not appearing in the onboarding form.

Reply:
oops yes thanks!